app/services/operations/complete_application.rb

```diff
@@ -24,14 +24,26 @@ def call # rubocop:disable Metrics/AbcSize
 
     private
 
-    # TODO: revert after user research
```

Comment:
I could not get the original implementation of this to work. There seemed to be a caching issue which caused flaky specs.